app/(aux)/terms.tsx

```
@@ -0,0 +1,90 @@
+                    {'\n\n'}- Violate any applicable laws or regulations
+                    {'\n'}- Impersonate any person or entity
+                    {'\n'}- Interfere with the proper functioning of the application
+                    {'\n'}- Attempt to gain unauthorized access to any part of the application
+                    {'\n'}- Use the application for any illegal or unauthorized purpose
+                </ThemedText>
+
+                <ThemedText style={[commonStyles.title, { fontSize: 18, marginBottom: 10 }]}>
+                    5. Disclaimer of Warranties
+                </ThemedText>
+                <ThemedText style={[commonStyles.text, { marginBottom: 20 }]}>
+                    Vested AI is provided "as is" without any warranties, either express or implied. We do not warrant that the application will be error-free, uninterrupted, or free of viruses or other harmful components.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)